src/menu/menu.c

```diff
@@ -1001,10 +1001,15 @@ update_client_list_combined_menu(struct server *server)
 
 		wl_list_for_each(view, &server->views, link) {
 			if (view->workspace == workspace) {
+				const char *tmp_title = view_get_string_prop(view, "title");
```

Comment:
Please could we just call this `title` rather than `tmp_title`.

Reply:
Ok, will change.